lib/util/serialization.js

```diff
@@ -73,6 +73,11 @@ module.exports = {
 		return getBinaryMiddleware().MEASURE_END_OPERATION;
 	},
 	get buffersSerializer() {
+		throw new Error(
+			"buffersSerializer was removed in favor of createBuffersSerializer"
+		);
+	},
```

Comment:
I think that's an unnecessary breaking change. Let's use the xxhash64 as default instead of throwing.